collections/_article/release-candidate-godot-4-2-rc-2.md

```diff
@@ -0,0 +1,101 @@
+- Export: iOS: Check if Xcode is installed in one-click deploy code ([GH-85168](https://github.com/godotengine/godot/pull/85168)).
+- GDExtension: iOS: Fix GDExtension init callback array reallocation ([GH-85216](https://github.com/godotengine/godot/pull/85216)).
+- GDScript: Fix lambda cross-thread dynamics (take 2) ([GH-85248](https://github.com/godotengine/godot/pull/85248)).
+- GUI: Add GraphEdit connection layer child as internal ([GH-85009](https://github.com/godotengine/godot/pull/85009)).
+- GUI: Fix crash when hiding a Control during mouse-entering ([GH-85284](https://github.com/godotengine/godot/pull/85284)).
+- Porting: Windows: Use mingw-std-threads in MinGW builds ([GH-85039](https://github.com/godotengine/godot/pull/85039)).
+- Rendering: Ensure 2D MSAA resolve is performed when 3D content but no 2D content in scene ([GH-84957](https://github.com/godotengine/godot/pull/84957)).
+- Rendering: Prevent crash in `_nvapi_disable_threaded_optimization` when attached to renderdoc ([GH-85121](https://github.com/godotengine/godot/pull/85121)).
+- Rendering: Avoid division by zero in the fix surface compatibility routine ([GH-85138](https://github.com/godotengine/godot/pull/85138)).
+- Rendering: Fix potential double-close of draw command label ([GH-85147](https://github.com/godotengine/godot/pull/85147)).
+- Rendering: Enable non-multiview advanced shader group whenever advanced shaders are requested ([GH-85194](https://github.com/godotengine/godot/pull/85194)).
+- Shaders: Make `AMOUNT_RATIO` constant in the shader language specification ([GH-85086](https://github.com/godotengine/godot/pull/85086)).
+- Shaders: Set some dialogs in the VisualShader editor to be exclusive ([GH-85205](https://github.com/godotengine/godot/pull/85205)).
+- Documentation and translation updates.
+
+This release is built from commit [`1ba920fad`](https://github.com/godotengine/godot/commit/1ba920fada9efc8c4476ded50fe673b8db541366).
+
+## Downloads
+
+{% include articles/download_card.html version="4.2" release="rc2" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- .NET build requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0), or [8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) installed in a standard location.
+- To export to Android, .NET 7.0 or later is required. To export to iOS, .NET 8.0 is required. Make sure to set the target framework in the `.csproj` file.
+
+{% include articles/prerelease_notice.html %}
+
+## Known issues
+
+We are aware of some regressions which still affect this release candidate, but which we may not be able to solve in time for the stable release. They're still tracked as high priority and should be fixed soon after the release with backports to 4.2.x maintenance releases.
```

Comment:
```suggestion
We are aware of some regressions which still affect this release candidate, but which we may not be able to solve in time for the stable release. They're still tracked as high priority and should be fixed soon in 4.2.x maintenance releases.
```
Although I know the development methodology (where you push to master and backport relevant things to older branches), I think it's clearer as I've suggested above.